rfcs/text/0024/ticket.yml

```diff
@@ -0,0 +1,151 @@
+---
+- name: ticket
+  group: 2
+  title: Ticket
+  short: Fields describing details of a ticket.
+  description: >
+    A ticket is defined as a documented record of any form of request or reported issue.
+    Tickets can be associated with hardware, software, access, procurement, etc. requests.
+    They can also be associated with ITIL /ITSM support processes
+    Ticket fields provide key basic details about the ticket itself.
+  footnote: >
+    Ticket types include, but are not limited to:
+    Alarm, Event, Incident, Problem, Case, Bug, Feature Request, Task, Epic, User Story,
+    Knowledge Article, Change Request, etc.
+  type: group
+
+  fields:
+
+    - name: assignee
+      level: extended
+      type: keyword
+      short: Assigned to name
+      description: >
+        Name of the person or group with whom the ticket is currently assigned.
+      example: Sector_7-G_Safety_Desk
+
+    - name: created
+      level: core
+      type: date
+      short: Timestamp the ticket was created
+      description: >
+        Timestamp the ticket was created.
+      example: 2021-03-09T17:05:23.773Z
+
+    - name: description.long
+      level: core
+      type: text
+      short: Long description of the ticket
+      description: >
+        Long description / body of the ticket.  Typically contains all details and
+        specifics of the ticket.
+      example: >
+        A fiber cut has occurred in Springfield due to a down telephone pole.  All internet and
+        IP related services are down diddly offline. Customer call in-rates are elevated.
+
+    - name: description.short
+      level: core
+      type: text
+      short: Short description of the ticket
+      description: >
+        Short description / title of the ticket.  Typically, a brief overview.
+      example: >
+        Fiber cut in Springfield - internet service down
+
+    - name: details
```

Comment:
Similar to `tags,` this field's function is described in a similar way to the top-level `labels` field. What's the motivation for duplicating that function underneath `tickets.*`?

Reply:
Same answer as `tags` I didn't think about reusing those fields until after the fact.  I copied their functionality originally, and then later realized they should just be reused.  I will remove.